packages/map-template/src/atoms/sessionTokenState.js

```diff
@@ -0,0 +1,8 @@
+import { atom } from 'recoil';
+
+const sessionTokenState = atom({
```

Comment:
Should we include Mapbox in the naming of this? It's only used for Mapbox.

Reply:
✅